tests/framework/shim/unix_shim.cpp

```diff
@@ -166,10 +174,19 @@ FRAMEWORK_EXPORT struct dirent* READDIR_FUNC_NAME(DIR* dir_stream) {
         std::vector<struct dirent*> folder_contents;
         std::vector<std::string> dirent_filenames;
         while (true) {
+            errno = 0;
             struct dirent* dir_entry = real_readdir(dir_stream);
+            if (errno != 0) {
```

Comment:
A similar thing could also be done in loader.c to avoid potential future confusion

Reply:
Yes, while I can't do much more than print an error message, its better than "oops no files!"